### ISSUE TITLE
compilation failed for 1.1.0

### ISSUE THREAD

**Issue by zh217** [Day8/re-frame-http-fx#5]
output below:
```
Failed to compile "resources/public/js/compiled/app.js" in 27.16 seconds.
----  Could not Analyze  resources/public/js/compiled/out/day8/re_frame/http_fx.cljs   line:63  column:79  ----

  No value supplied for key: []

  62  (s/def ::request-map (s/keys :req-un [::method ::uri ::response-format ::on-success ::on-failure]))
  63  (s/def ::sequential-or-map (s/or :request-map ::request-map :seq-request-maps (s/coll-of ::request-map [])))
                                                                                    ^--- No value supplied for key: []
  64
  65  (reg-fx
  66    :http-xhrio

----  Analysis Error : Please see resources/public/js/compiled/out/day8/re_frame/http_fx.cljs  ----
```

**Comment by danielcompton** [comment]
Hi @zh217 I can't reproduce this on our projects. Can you share:

* How you're compiling this
* Your build config
* What your dependencies are, particularly Clojure and ClojureScript versions
* A minimal reproduction example

As a wild guess, I'd guess you're using an older version of ClojureScript which doesn't support/understand this version of `coll-of`?

**Comment by rsnara** [comment]
Strange... I'm running into the exact same problem. The project I'm working on is located here: https://github.com/uwboys/palr-web/tree/bug/reframe-http-fx.

**Comment by danielcompton** [comment]
> As a wild guess, I'd guess you're using an older version of ClojureScript which doesn't support/understand this version of coll-of?

Well I was sort of right here, `re-frame-http-fx` supported the older `coll-of` syntax. I've bumped the deps and fixed the syntax to work with CLJS 1.9.229. That should fix it, let me know if you still have issues though.
